batch/Makefile

```diff
@@ -92,6 +98,8 @@ test-local: batch-secrets/batch-test-cloud-sql-config.json test/jwt-test-user-to
 # local means server and test client are two processes on one machine
 # in-cluster means in a k8s pod (from which we get k8s creds)
 test-local-in-cluster: /batch-secrets/batch-test-cloud-sql-config.json
+test-local-in-cluster: /batch-test-gsa-key/privateKeyData
+test-local-in-cluster: /batch-test-jwt/jwt
```

Comment:
this still seems wrong to me. There are no rules to generate these dependencies, right? It seems really odd to include them here. We don't include make dependencies on, say, the C++ standard library at `/usr/lib/libstdc++.so`

Reply:
Fixed these by removing them